inttest/ctr/singlenode_test.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package ctr
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/k0sproject/k0s/inttest/common"
+	"github.com/stretchr/testify/suite"
+)
+
+type CtrSuite struct {
+	common.FootlooseSuite
+}
+
+func (s *CtrSuite) TestK0sCtrCommand() {
+	ssh, err := s.SSH(s.ControllerNode(0))
+	s.Require().NoError(err)
+	defer ssh.Disconnect()
+
+	_, err = ssh.ExecWithOutput("k0s install controller --enable-worker")
+	s.Require().NoError(err)
+
+	_, err = ssh.ExecWithOutput("rc-service k0scontroller start")
```

Comment:
If you change this to `k0s start` we can test the #896 at the same time :)

```suggestion
	_, err = ssh.ExecWithOutput("k0s start")
```

Reply:
Thanks. You're right, but the command `re-service k0scontroller start/stop` is also used in other tests, so we could change all of them to `k0s start/stop` in another pull request, because it's not related to the issue. What do you think?

Reply:
I'd rather that we don't touch other tests. Let's scope this PR to ctr only.

Reply:
Ok, done